gen_test_vectors.py

```diff
@@ -186,21 +195,20 @@ def vector7():
     presig = neg_parity + presig[1:]
 
     # check if flipping R'.y also flips the y-coordinate of adaptor
-    T1 = cpoint(adaptor)
     extracted_adaptor = schnorr_extract_adaptor(msg, pubkey, presig)
+    assert extracted_adaptor is not None
+    T1 = cpoint(adaptor)
     T2 = cpoint(extracted_adaptor)
     assert (adaptor != extracted_adaptor)
     assert (T1 == point_negate(T2))
 
-    return (None, pubkey, None, msg, adaptor, presig, "FALSE", "negated presig[0:33] value")
-
-# todo: add test vector containing invalid parity byte (like 4, 5) for R'
+    return (None, pubkey, None, msg, adaptor, presig, "FALSE", "The first byte of presig (parity byte) is negated")
```

Comment:
The byte isn't really negated. Maybe "The LSB of the first byte of presig (parity byte) is flipped".

Reply:
Fixed in a777151